### PR TITLE
updated leftover quay links

### PR DESCRIPTION
**Pull request by scoyle391**

### CODE REVIEW

File: reference/running.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
         service: ambassador
     spec:
       containers:
-        image: quay.io/datawire/ambassador:0.50.0
+        image: quay.io/datawire/ambassador:$version$
```
Comment (LukeShu) [inline reference/running.md]: Should this be `aes` instead of `ambassador`?

File: user-guide/with-istio.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ spec:
       serviceAccountName: ambassador
       containers:
       - name: ambassador
-        image: quay.io/datawire/ambassador:0.50.1
+        image: quay.io/datawire/ambassador:$version$
```
Comment (LukeShu) [inline user-guide/with-istio.md]: Should this be `aes` instead of `ambassador`?

File: user-guide/early-access.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ Looking for early access to the Ambassador Edge Stack? [Click here to get starte
 
 From time to time, the Ambassador Edge Stack may ship early access releases to test major changes. **Early access releases are not supported for production use** but are intended to gain early feedback from our community before shipping a release.
 
-Early access releases will always have names that include the string "-ea" followed by a build number. For example, `0.50.0-ea1` is the first early access build of the Ambassador Edge Stack 0.50.0.
+Early access releases will always have names that include the string "-ea" followed by a build number.
 
 ## Early Access Status
 
-There are currently no early access releases available.
+The current early access build for the Ambassador Edge Stack is `1.0.0-ea1`.
```
Comment (LukeShu) [inline user-guide/early-access.md]: You should spell `1.0.0-ea1` as `$version$`, so it doesn't need replaced in multiple places.

File: user-guide/early-access.md [hunk 4]
```diff
@@ -6,17 +6,17 @@ Looking for early access to the Ambassador Edge Stack? [Click here to get starte
...
 
 ### Installing the Ambassador Edge Stack Early Access releases
 
-Use a Kubernetes deployment as usual, but use the image `quay.io/datawire/ambassador:0.50.0-ea5`.
+Use a Kubernetes deployment as usual, but use the image `quay.io/datawire/aes:1.00.0-ea1`.
```
Comment (LukeShu) [inline user-guide/early-access.md]: Like above, you should write this as `$version$`, but also there's an extra `0` in there.

File: user-guide/early-access.md (hunk 3 above)
Comment (scoyle391) [reply user-guide/early-access.md]: Fixed the incorrect numbering. But to your other point, isn't the point of this page showing users how to access an early release? $version$ is just the latest release.

File: user-guide/early-access.md (hunk 4 above)
Comment (scoyle391) [reply user-guide/early-access.md]: same comments as above.

File: user-guide/early-access.md (hunk 3 above)
Comment (LukeShu) [reply user-guide/early-access.md]: There are separate EA and GA `versions.yml` files; the EA one should have the EA version.  If that isn't currently the case, we should update the `versions.yml` file.

Comment (scoyle391) [reply user-guide/early-access.md]: fixed

File: user-guide/early-access.md (hunk 4 above)
Comment (scoyle391) [reply user-guide/early-access.md]: fixed

**Pull request merged**